cub/cub/block/block_merge_sort.cuh

```diff
@@ -211,6 +211,136 @@ private:
 
   const unsigned int linear_tid;
 
+  template <typename CompareOp,
+            bool IS_STABLE,
+            bool IS_LAST_TILE = true>
+  _CCCL_DEVICE _CCCL_FORCEINLINE void SortCommon(KeyT (&keys)[ITEMS_PER_THREAD],
+                                       ValueT (&items)[ITEMS_PER_THREAD],
+                                       CompareOp compare_op,
+                                       int valid_items,
+                                       KeyT oob_default)
+  {
+    if (IS_LAST_TILE)
```

Comment:
I assume this is the one place you mentioned where `CUB_IF_CONSTEXPR` is having issues?

Reply:
Yes.